src/cirrus/cirrus_setup.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+"""
+selfsetup command to prompt for/configure cirrus post install
+"""
+import os
+import sys
+import json
+import getpass
+import requests
+
+from argparse import ArgumentParser
+
+from cirrus.configuration import load_configuration, get_creds_plugin
+from cirrus.logger import get_logger
+
+LOGGER = get_logger()
+GITHUB_AUTH_URL = "https://api.github.com/authorizations"
```

Comment:
Im going to nuke bootstrap once the selfsetup command is working. Should probably make the GH URL configurable/overrideable somehow, thats probably a different thing tho